Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html

```diff
@@ -4,15 +4,31 @@ <h1> Elysia Range Simulator </h1>
     <form name="simulator" #simulator="ngForm" novalidate>
       <label>Speed (km/h):</label>
       <br>
-      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)">
-      <br>
+      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)" max=90 min=0>
+      <div *ngIf="speedError == 1; else speedError2" class="errorMessage">Speed is required</div>
+      <br/>
```

Comment:
The closing tag for a page break is not needed in HTML so you can remove the slash from the tag.